.github/workflows/no-cache-multidim-interop.yml

```diff
@@ -0,0 +1,19 @@
+on:
+  workflow_dispatch:
+  pull_request:
+  push:
+    branches:
+      - "master"
```

Comment:
We share the same triggers with the other workflow right? Why not have it in the other file as a separate job?

Reply:
I wanted it to run in parallel and isolated. Is that the case with a separate job?

Reply:
Yes jobs run in parallel in a workflow unless you declare a dependency with `needs:`